monei.php

```
@@ -1986,6 +2267,108 @@ public function hookPaymentOptions($params)
+
+        self::$captureInProgress = true;
+
+        try {
+            // ⚠️ Read the amount with a plain query rather than through the
+            // Doctrine repository. This hook has to work in every context that can
+            // move an order — cron, an ERP sync, the webservice API, a CLI script —
+            // and several of those bootstrap PrestaShop without the module's
+            // service container, where getRepository() is null and this would be a
+            // fatal in the middle of a merchant's order flow.
+            $amount = (int) Db::getInstance()->getValue(
+                'SELECT amount FROM ' . _DB_PREFIX_ . 'monei2_payment WHERE id_order = ' . (int) $order->id
```

PHPStan (2.2.8)

[error] Constant _DB_PREFIX_ not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)